fvwm/infostore.c

```diff
@@ -145,12 +145,13 @@ void CMD_InfoStoreAdd(F_CMD_ARGS)
 		goto error;
 	value = fxstrdup(token);
 
+	insert_metainfo(key, value);
+
 error:
 	if (key == NULL || value == NULL) {
 		fvwm_debug(__func__, "Bad arguments given.");
 		goto out;
 	}
-	insert_metainfo(key, value);
 out:
```

Comment:
Shouldn't this just be

```
error:
	if (key == NULL || value == NULL)
		fvwm_debug(__func__, "Bad arguments given.");
```

now, no need for the `out:` label.

Reply:
We can just fall-through now, yes.